src/dataflow/operators/to_stream.rs

```diff
@@ -4,7 +4,7 @@ use progress::Timestamp;
 
 use Data;
 use dataflow::channels::Content;
-use dataflow::operators::operator::source;
+use dataflow::operators::generic::operator::source;
```

Comment:
Ah, it is re-exported in `generic`. I used the longest form possible in all of the examples, under the (possibly misguided) theory that (i) they can easily be corrected if changed, (ii) they test that others like them might break, and (iii) they show others where stuff actually exists.